### PR TITLE
utils_misc: Add a new func for get host_bridge on a host

### PR DESCRIPTION
**Pull request by xiaodwan**
Add a new function to get host bridges on a host

### CODE REVIEW

**Comment by xiaodwan** [comment]
@dzhengfy @chunfuwen Please help to review this PR, thanks.

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -634,6 +634,23 @@ def get_full_pci_id(pci_id):
         return None
 
 
+def get_host_bridge_id(session=None):
```
Comment (dzhengfy) [inline virttest/utils_misc.py:637]: could you help move this function to utils_sys.py? We avoid adding new function into utils_misc.py.

**Review by dzhengfy — COMMENTED** [review]
Codes LGTM

File: virttest/utils_misc.py (hunk 1 above)
Comment (chloerh) [reply virttest/utils_misc.py]: +1

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**